utils/email/index.js

```diff
@@ -0,0 +1,79 @@
+import path from 'path';
+import fs from 'fs';
+import sendgridEmail from '@sendgrid/mail';
+import mjml2html from "mjml";
+
+if (process.env.SENDGRID_API_KEY) {
+  sendgridEmail.setApiKey(process.env.SENDGRID_API_KEY);
+}
+
+
+function sendEmail({recipient, subject, htmlBody, textBody}) {
+  if (!process.env.SENDGRID_API_KEY) {
+    console.error('SendGrid error: SENDGRID_API_KEY is not set');
+    return;
+  }
+  const from = 'mentors@tupu.io'; // Change to your verified sender
+  recipient = process.env.DEV_EMAIL_RECIPIENT || recipient;
+  subject = buildSubjectLine(subject);
+  const msg = {
+    to: recipient,
+    from: process.env.DEV_EMAIL_RECIPIENT || from, 
+    subject: subject,
+    text: textBody,
+    html: htmlBody,
+  };
+  sendgridEmail
+    .send(msg)
+    .then(() => {
+      console.log(`Email sent to SendGrid for <${recipient}>,\n subject "${subject}"`);
+    })
+    .catch((error) => {
+      console.error('SendGrid error:', error)
+      console.error(`Could not deliver email to <${recipient}>,\n subject "${subject}"`);
+    });
+}
+
+const buildSubjectLine = (subject) => {
+  const dev_prefix = process.env.DEV_EMAIL_RECIPIENT ? "[DEBUG] " : "";
+  const prefix = "[TUPU] ";
+  return `${dev_prefix}${prefix}${subject}`;
+}
+
+const resolveTemplatesPath = () => { 
+  return path.resolve(process.cwd(), 'utils', 'email', 'templates');
+}
+
+const loadHTMLTemplate = (templateName) => {
+  const templatesPath = resolveTemplatesPath();
+  const templateFilename = path.join(templatesPath, `${templateName}.mjml`);
+  const mjmlTemplateContent = fs.readFileSync(templateFilename, 'utf8');
```

Comment:
We need to check if this will work in Netlify functions. It should be [possible](https://www.netlify.com/blog/2021/08/12/how-to-include-files-in-netlify-serverless-functions/) but it will probably not work out of the box because netlify doesn't know that those files are required for which functions.

I think we need in `netlify.toml`, something like:

```
[functions]
  included_files = ["utils/email/templates/*"]
```

Can you add it to this PR, please? Then I can try it in the preview.

Reply:
done

Reply:
I have not thought about this, thanks a lot for noticing and suggesting the solution